EngineeringModel/ViewModels/ElementDefinitionBrowser/ElementDefinitionsBrowserViewModel.cs

```diff
@@ -354,6 +375,9 @@ protected override void InitializeCommands()
 
             this.HighlightElementUsagesCommand = ReactiveCommand.Create();
             this.HighlightElementUsagesCommand.Subscribe(_ => this.ExecuteHighlightElementUsagesCommand());
+
+            this.CopyModelCodeToClipboardCommand = ReactiveCommand.Create(this.WhenAnyValue(x => x.CanCopyToClipboard));
```

Comment:
I think you can just do `Create()` because you never disable this command. Then `CanCopyToClipboard` is not needed.